pages/index.js

```diff
@@ -1,102 +1,174 @@
+const GITHUB_BRANCH = "main"; /* optional: defaults to default branch */
+
+const TOKEN = process.env.ACCESS_TOKEN; // create token at https://github.com/settings/tokens/new?scopes=repo
 
 const useStyles = makeStyles((theme) => ({
   paper: {
     marginTop: theme.spacing(8),
-    display: 'flex',
-    flexDirection: 'column',
-    alignItems: 'center',
+    display: "flex",
+    flexDirection: "column",
+    alignItems: "center",
   },
   avatar: {
     margin: theme.spacing(1),
     backgroundColor: theme.palette.secondary.main,
   },
   form: {
-    width: '100%', // Fix IE 11 issue.
+    width: "100%", // Fix IE 11 issue.
     marginTop: theme.spacing(1),
   },
   submit: {
     margin: theme.spacing(3, 0, 2),
-  }
-
+  },
 }));
 
 const theme = createMuiTheme({
   overrides: {
     MuiButton: {
       root: {
-        background: 'linear-gradient(45deg, #FE6B8B 30%, #FF8E53 90%)',
+        background: "linear-gradient(45deg, #FE6B8B 30%, #FF8E53 90%)",
         borderRadius: 3,
         border: 0,
-        color: 'white',
+        color: "white",
         height: 48,
-        padding: '0 30px',
-        boxShadow: '0 3px 5px 2px rgba(255, 105, 135, .3)',
+        padding: "0 30px",
+        boxShadow: "0 3px 5px 2px rgba(255, 105, 135, .3)",
       },
     },
     MuiCheckbox: {
       root: {
-        padding: '1px 9px'
-      }
-    }
+        padding: "1px 9px",
+      },
+    },
   },
 });
 
-const MyFormTemplate = (props) => <FormTemplate {...props} showFormControls={false} />
+const MyFormTemplate = (props) => (
+  <FormTemplate {...props} showFormControls={false} />
+);
 
 const validate = (values) => {
-  console.log(values)
+  console.log(values);
   const errors = {};
 
-  for(let i=1; i<=17; i++){
-    if (values.SDGs && values.SDGs.includes(i) && ! values['evidenceText'+i] && ! values['evidenceURL'+i]) {
-      errors['evidenceText'+i] = 'Either the description or a URL is required';
-      errors['evidenceURL'+i] = 'Either the description or a URL is required';
+  for (let i = 1; i <= 17; i++) {
+    if (
+      values.SDGs &&
+      values.SDGs.includes(i) &&
+      !values["evidenceText" + i] &&
+      !values["evidenceURL" + i]
+    ) {
+      errors["evidenceText" + i] =
+        "Either the description or a URL is required";
+      errors["evidenceURL" + i] = "Either the description or a URL is required";
     }
   }
 
   return errors;
+};
+
+function openPR(values) {
+  let sdgNumber, evidenceText;
+  for (let i = 0; i < values.SDGs.length; i++) {
+    sdgNumber = JSON.stringify(values.SDGs[i]);
+    sdgNumber = parseInt(sdgNumber);
+    evidenceText = "evidenceText".concat(sdgNumber); //
+
+    values.SDGs[i] = {
+      SDGNumber: sdgNumber,
+      evidenceText: values[evidenceText],
+    };
+
+    delete values[evidenceText];
+  }
+
+  const myJSON = JSON.stringify(values, null, "\t");
+
+  // Return project json file aligned with naming convention
+  let name = values.name.toLowerCase().replace(/ /g, "_") + ".json";
```

Comment:
I know that you have taken this from other existing files, but this is not enough. We need to replace characters like `á` for `a` and the like.

This line written in bash has the expanded replacement that we would need to port to Javascript. Would you mind taking care of this?

https://github.com/unicef/publicgoods-candidates/blob/ecd742361eb42e74208948d111d00c473ec22bcc/scripts/check-filenames.bash#L14

Reply:
Resolved in commit 5740843.